hack/tools/scripts/update_trusted_task_bundles.py

```
@@ -489,6 +492,163 @@ def print_summary(
+        List of (version, script_url) for versions that have migration scripts.
+    """
+    # Strip "task-" prefix for the build-definitions repo path
+    short_name = task_name.removeprefix("task-")
+
+    versions = get_intermediate_versions(current_version, target_version)
+    available = []
+
+    for version in versions:
+        url = MIGRATION_SCRIPT_URL.format(task_name=short_name, version=version)
+        try:
+            result = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)